deploy/Chart/templates/dynamic-rp/configmaps.yaml

```diff
@@ -35,6 +35,8 @@ data:
       port: 6062
     secretProvider:
       provider: kubernetes
+    kubernetes:
+      kind: default
```

Comment:
This is only needed for dynamic-rp. UCP doesn't talk directly to Kubernetes, and applications-rp doesn't have the same expectations for testing.